lib/static.js

```diff
@@ -0,0 +1,64 @@
+const fs = require('fs');
+const path = require('path');
+const mime = require('mime');
+const { Readable } = require('stream');
+const bootstrap = require('./shared/bootstrap');
+const gen = require('./shared/gen');
+const { getDataPlaceholder } = require('./shared/utils');
+
+module.exports = bootstrap(async function(options, config) {
+    return new Promise(async resolve => {
+        const { model } = options;
+
+        const genDataStream = await gen['/data.json'](config.models[0], options);
+
+        let genData = '';
+
+        genDataStream.on('data', data => {
+            genData += data;
+        });
+
+        genDataStream.on('end', () => {
+            const html = `build/${model}/index.html`;
```

Comment:
Suppose `options.whatever` should be instead of `build/` prefix

Reply:
Sure, fixed